stable/openvpn/templates/config-openvpn.yaml

```diff
@@ -163,8 +153,15 @@ data:
       sed 's|NETWORK|'"${NETWORK}"'|' -i /etc/openvpn/openvpn.conf
       sed 's|NETMASK|'"${NETMASK}"'|' -i /etc/openvpn/openvpn.conf
 
-      # exec openvpn process so it receives lifecycle signals
-      exec openvpn --config /etc/openvpn/openvpn.conf
+      if [ "{{ .Values.debug }}" == "true" ]; then
+          echo ">>> DEBUG: printing openvpn.conf below"
+          echo ========== /etc/openvpn/openvpn.conf ==========
+          cat /etc/openvpn/openvpn.conf
+          echo ===============================================
+      fi
+
+      openvpn --config /etc/openvpn/openvpn.conf
```

Comment:
we should keep the `exec` so openvpn receives signals sent by k8s (otherwise openvpn will always wait for its graceful termination period when performing a rolling upgrade for example)

Reply:
Cool, done in latest push